exports/utils.go

```diff
@@ -0,0 +1,56 @@
+package exports
+
+import (
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/redhatinsights/export-service-go/models"
+)
+
+const (
+	formatDate     = "2006-01-02"
+	formatDateLen  = 10
+	formatDateTime = time.RFC3339
+)
+
+func initQuery(q url.Values) (result models.QueryParams, err error) {
+	// all params are optional
+	result.Name = q.Get("name")
+	result.Application = q.Get("application")
+	result.Resource = q.Get("resource")
+	result.Status = q.Get("status")
+
+	created := q.Get("created")
+	expires := q.Get("expires")
+
+	// created and expires should be date only, not date-time strings
+	if created != "" {
+		result.Created, err = parseDate(created)
+
+		if err != nil {
+			return models.QueryParams{}, fmt.Errorf("'%s' is not a valid date in ISO 8601", expires)
+		}
+	}
+
+	if expires != "" {
+		result.Created, err = parseDate(expires)
+		if err != nil {
+			return models.QueryParams{}, fmt.Errorf("'%s' is not a valid date in ISO 8601", expires)
```

Comment:
Maybe add `(YYYY-MM-DD)` here?
Also, in testing, I'm seeing:
```
{
    "message": "'' is not a valid date in ISO 8601",
    "code": 400
}
```
It doesn't look like it's grabbing the incorrect string from the request, it's just "" in the error.